op-challenger/README.md

```diff
@@ -1,48 +1,38 @@
 # op-challenger
 
-The `op-challenger` is a modular **op-stack** challenge agent
-written in golang for dispute games including, but not limited to, attestation games, fault
-games, and validity games. To learn more about dispute games, visit the
-[fault proof specs](../specs/fault-proof.md).
+The `op-challenger` is a modular **op-stack** challenge agent written in
+golang for dispute games including, but not limited to,attestation games,
+fault games, and validity games. To learn more about dispute games, visit
+the [fault proof specs][proof-specs].
 
-## Quickstart
-
-First, clone this repo. Then run:
-
-```shell
-cd op-challenger
-make alphabet
-```
+[proof-specs]: https://specs.optimism.io/experimental/fault-proof/index.html
 
-This creates a local devnet, starts a dispute game using the simple alphabet trace type and runs two op-challenger
-instances with differing views of the correct alphabet to play the game.
+## Quickstart
 
-Alternatively, you can build the `op-challenger` binary directly using the pre-configured
-[Makefile](./Makefile) target by running `make build`, and then running `./bin/op-challenger --help`
-to see a list of available options.
+To build the `op-challenger`, run `make` (which executes the `make build`
+[Makefile](./Makefile) target). To view a list of available commands and
+options, run `./bin/op-challenger --help`.
 
 ## Usage
 
-`op-challenger` is configurable via command line flags and environment variables. The help menu
-shows the available config options and can be accessed by running `./op-challenger --help`.
```

Comment:
This seems like a useful paragraph to keep to push people towards `--help` to find the full usage info.